.storybook/main.js

```diff
@@ -0,0 +1,29 @@
+const { aliases } = require('../package.json');
+const path = require('path');
+
+module.exports = {
+    stories: ['../stories/**/*.stories.@(js|mdx|tsx|ts|jsx)'],
+    addons: ['@storybook/addon-docs', '@storybook/addon-viewport', '@storybook/addon-knobs/', '@storybook/addon-links/', '@storybook/addon-actions/', '@storybook/addon-toolbars', '@storybook/addon-a11y', '@storybook/addon-controls', '@storybook/addon-backgrounds'],
+    webpackFinal: async (config, { configType }) => {
```

Comment:
I believe storybook has a recipe for this now, should look into that

Reply:
I think your referring to essentials?

Reply:
nvm they got rid of it in this major version...

Reply:
nvm i found it, its a preset mb
https://www.npmjs.com/package/@storybook/preset-typescript

Reply:
maybe storybook is more "developed", but i have a feeling the build stuff would impode for some reason

Reply:
@fpunny Storybook comes with TypeScript configuration built in now? Is that still needed?